src/components/SwapWidget/LimitOrder/index.tsx

```diff
@@ -525,6 +532,9 @@ const LimitOrder: React.FC<Props> = ({ swapType, setSwapType, isLimitOrderVisibl
           <Box width="100%" mt={10}>
             {renderButton()}
           </Box>
+          <Box width="100%" mt={10} display="flex" justifyContent="center">
+            <SettingsText onClick={() => setOpenSettings(true)}>Settings</SettingsText>
```

Comment:
Now as I think, limit order should not have settings because its ultimately different thing as we are using galeto library. let me know if I am thinking differently.

Reply:
Removed